src/main/java/seedu/address/model/applicant/Age.java

```diff
@@ -17,7 +17,7 @@
     public Age(String age) {
         requireNonNull(age);
         checkArgument(isValidAge(age), MESSAGE_CONSTRAINTS);
-        this.age = age;
+        this.value = age;
```

Comment:
Would leaving it as age be more intuitive for developers?

Otherwise, maybe you could consider this workaround?
```
public Age(String value) {
        requireNonNull(value);
        checkArgument(isValidAge(value), MESSAGE_CONSTRAINTS);
        this.value = age;
}
```